### ISSUE TITLE
WEB

### ISSUE THREAD

**Issue by MMAKENPO** [thedonat/iTunes-Search-App#1]
HELLO BRO!!!TELL ME PLEASE WE CAN SEARCH SONGS AND ALBUMS FROM ITUNES?AND IF YES,PLEASE COULD YOU TELL ME THE WEB?

**Comment by thedonat** [comment]
Yes, you can search any word on search bar, it shows the results for music, movies and podcasts.
I've used Apple's iTunes Seach API. Here is the website: 
https://developer.apple.com/library/archive/documentation/AudioVideo/Conceptual/iTuneSearchAPI/Searching.html#//apple_ref/doc/uid/TP40017632-CH5-SW1

**Comment by MMAKENPO** [comment]
WOOWWWWWW....MY FRIEND CAN YOU MAKE ONE WEB PLEASE???I PRESIATE IT SO MUCHHHHHHH!!!I AM NOT DEVELOPMENT AND I HAVE NO IDEA HOW TO DO IT!!!CAN YOU???
